ui/src/shared/components/DeleteDataForm/BucketsDropdown.tsx

```diff
@@ -4,10 +4,13 @@ import {connect} from 'react-redux'
 import {SelectDropdown} from '@influxdata/clockface'
 
 // Types
-import {AppState, Bucket} from 'src/types'
+import {AppState} from 'src/types'
+
+// Selectors
+import {sortBucketNames} from 'src/buckets/selectors/index'
 
 interface StateProps {
-  buckets: Bucket[]
+  buckets: string[]
```

Comment:
i think you should rename this to `bucketNames` since that's now what it represents, right? otherwise it should probably stay a `Bucket[]`

Reply:
good point